requirements.txt

```diff
@@ -1,23 +1,23 @@
-aio-pika==8.1.1
-fastapi==0.83.0
-gunicorn==20.0.4
-httptools==0.6.0
-httpx==0.19.0
-jsonschema==3.2.0
-neo4j==5.0.1
-pamqp==3.2.0
-pydantic==1.10.2
-pytest==7.1.2
-pytest-asyncio==0.15.1
+aio-pika==9.4.2
+fastapi==0.111.1
+gunicorn==22.0.0
+httptools==0.6.1
+httpx==0.27.0
+jsonschema==4.23.0
+neo4j==5.22.0
+pamqp==3.3.0
+pydantic<=2.8.2
```

Comment:
This is going to conflict with reasoner-pydantic, which requires that pydantic be pinned to v1.

Reply:
Good to know!  I'd do the same for AC requirements.